src/parser/xasl_generation.c

```diff
@@ -3250,6 +3419,32 @@ pt_is_spec_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *void_arg, int *c
   return tree;
 }
 
+/*
+ * pt_check_hashable () - check whether hashable or not
+ *   return:
+ *   parser(in):
+ *   tree(in):
+ *   void_arg(in):
+ *   continue_walk(in):
+ */
+static PT_NODE *
+pt_check_hashable (PARSER_CONTEXT * parser, PT_NODE * tree, void *void_arg, int *continue_walk)
+{
+  *continue_walk = PT_CONTINUE_WALK;
```

Comment:
continue_walk is guaranteed as "not null"?

Reply:
yes. it is initialized in parser_walk_tree().